python/cudf/cudf/io/csv.py

```diff
@@ -59,14 +59,14 @@ def read_csv(
 
     return libcudf.csv.read_csv(
         filepath_or_buffer,
-        lineterminator=lineterminator,
-        quotechar=quotechar,
+        lineterminator=str(lineterminator),
```

Comment:
Similar to how we now handle `compression`, should we push the conversion to `str` down to the `libcudf.csv.read_csv` level? Ditto for all the others.

Reply:
Sure, Done 👍